### DISCUSSION TITLE
fatal error: SPI.h: No such file or directory

### DISCUSSION THREAD

**Discussion by ceakins** [Bodmer/TFT_eSPI#1041] [Compatible support libraries]
Added this library to my platformio project, but haven't written code to use it, compiler is complaining about that missing file. What other library do I need to install. I'm going to be doing parallel ILI9341 based screen.

**Comment by Bodmer** [comment]
You need to load the Arduino core support environment. You should find help on the PlatformIO web page. Users have successfully used platformIO with the library but this is not something I can help with.
